src/views/v2/list/nav/next.php

```diff
@@ -22,13 +22,15 @@
 		class="tribe-events-c-nav__next tribe-common-b2 tribe-common-b1--min-medium"
 		data-js="tribe-events-view-link"
 	>
-		<?php
-			$events_label = '<span class="tribe-events-c-nav__next-label-plural"> ' . tribe_get_event_label_plural() . '</span>';
-			echo wp_kses(
-				/* translators: %s: Event (plural or singular). */
-				sprintf( __( 'Next %1$s', 'the-events-calendar' ), $events_label ),
-				[ 'span' => [ 'class' => [] ] ]
-			);
-		?>
+		<span class="tribe-events-c-nav__next-label">
+			<?php
+				$events_label = '<span class="tribe-events-c-nav__next-label-plural">' . tribe_get_event_label_plural() . '</span>';
+				echo wp_kses(
+					/* translators: %s: Event (plural or singular). */
+					sprintf( __( 'Next %1$s', 'the-events-calendar' ), $events_label ),
+					[ 'span' => [ 'class' => [] ] ]
```

Comment:
* :no_entry_sign: Short array syntax is not allowed
* :no_entry_sign: Short array syntax is not allowed
* :no_entry_sign: Short array syntax is not allowed

<sub>hash: ce71950bfcca3c28dd9531432015c26d</sub>